python/pyplugin_installer/installer_data.py

```diff
@@ -102,8 +102,7 @@
 settingsGroup = "app/plugin_installer"
 reposGroup = "app/plugin_repositories"
 
-officialRepo = (QCoreApplication.translate("QgsPluginInstaller", "QGIS Official Plugin Repository"), "https://plugins.qgis.org/plugins/plugins.xml")
-
+officialRepo = (QCoreApplication.translate("QgsPluginInstaller", "QGIS Official Plugin Repository"), QgsSettings().value("qgis/officialpluginrepo", "https://plugins.qgis.org/plugins/plugins.xml"))
```

Comment:
Shouldn't that rather be an option to disable the official plugin repository, so a "Kadas Albireo Plugin Repository" can be added?

Reply:
Yeah, agreed...